packages/contracts-bedrock/scripts/print-addresses.sh

```diff
@@ -42,6 +42,7 @@ for file in "$DIR"/*.json; do
     filename=$(basename "$file" .json)
 
     # If SDK mode is on and the filename is not in the list, skip it
+    # shellcheck disable=SC2199,SC2076
     if $SDK_MODE && [[ ! " ${SDK_FILES[@]} " =~ " ${filename} " ]]; then
```

Comment:
[SC2199](https://www.shellcheck.net/wiki/SC2199) is "Arrays implicitly concatenate in `[[ ]]`. Use a loop (or explicit `*` instead of `@`)", and [SC2076](https://www.shellcheck.net/wiki/SC2076) is "Don't quote rhs of `=~`, it'll match literally rather than as a regex".

I've disabled SC2076 since it seems we want to match literally here, and I disabled SC2199 since concatenating the array contents seems ok for this use case and keeps the code more concise than the recommended `for` loop refactor. I don't feel strongly here and can refactor to the loop if we prefer